docs/docs-beta/docusaurus.config.ts

```diff
@@ -25,6 +25,10 @@ const config: Config = {
     require.resolve('docusaurus-plugin-image-zoom'),
   ],
   themeConfig: {
+    announcementBar: {
+      id: 'announcementBar',
+      content: `<b>You are viewing a preview of our new docs site. If you have any feedback, please let us know on <a target="_blank" href="https://github.com/dagster-io/dagster/issues/new/choose">GitHub</a>. If you are looking for the legacy docs, go <a target="_blank" href="https://docs.dagster.io/">here</a>!</b>`,
```

Comment:
should we maybe make a dedicate discussion category (e.g. Docs Beta Feedback) or label for this, so we can be more on top of feedback coming from here?

Reply:
also, when this is still in beta, should we refer the status quo to "current" instead of "legacy"?

Reply:
Good point, I will update to "current", and I also agree with using a specific discussion.

Reply:
For now I will point to:

https://github.com/dagster-io/dagster/discussions/23031

So that I don't have to create a new discussion.

Reply:
Updated!